Modules/OrgUnit/_PublicApi/OrgUnitUserService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace OrgUnit\_PublicApi;
```

Comment:
Does the `_` has a special meaning here?

Reply:
No; i removed that.